### PR TITLE
Fixing test errors and warnings

### PR DESCRIPTION
**Pull request by SiddhantSadangi**
# Description

Code in scope:
* Optuna examples
* Presenting cross validation results with neptune.ai

__Note__: Some tests on python 3.11 might fail as [Optuna](https://pypi.org/project/optuna/) and [PyTorch](https://github.com/pytorch/pytorch/issues/86566) do not support 3.11 yet.

__Related to:__ `fix tests errors`

---

## ❔ This change

- [ ] adds a new feature
- [X] fixes breaking code
- [X] is cosmetic (refactoring/reformatting)

---

## ✔️ Pre-merge checklist

Mark which items have been completed:

- [X] Refactored code ([sourcery](https://sourcery.ai/))
- [X] Tested code locally
- [ ] Added code to GitHub tests ([notebooks](workflows/test-notebooks.yml), [scripts](workflows/test-scripts.yml))
- [ ] Updated GitHub [README](../README.md)

### 🪝 Run the following to both [pre-commit](https://pre-commit.com/) hooks and JupyterLab (turn on `format on save`)

- [X] Stripped outputs from notebooks - ([nbstripout](https://pypi.org/project/nbstripout/))
- [X] Sorted imports - ([isort](https://pycqa.github.io/isort/))
- [X] Formatted code - ([back](https://github.com/psf/black))

### 📄 Update documentation

Coordinate with @normandy7

- [ ] Update content in the [docs](https://docs.neptune.ai)
- [ ] (For new content) Add links to examples [README](../README.md)

---

## 🧪 Test Configuration

- OS: Windows 11
- Python version: 3.10.8
- Neptune version: 0.16.9
- Affected libraries with version: neptune-optuna==0.9.13, optuna==3.0.4

---

## ✔️ Post-merge checklist

These tasks should be done after the code is merged to main. Create a subtask under the main ClickUp task to have these on your radar and share the subtask here.

__Post-merge subtask:__ `(ClickUp/JIRA task name)`

- [ ] Update relevant blogs with impacted code/functionality (Contact @Patrycja-J for the list of blogs)
- [ ] Add functionality under the Neptune column of the `competitor comparison sheet` (Coordinate with @SiddhantSadangi)
- [ ] Update integration README (Contact @Patrycja-J)

### CODE REVIEW

**Comment by SiddhantSadangi** [comment]
@normandy7 
Once this is merged, we'll need a few changes in the [Optuna integration docs](https://docs.neptune.ai/integrations/optuna/):
1. 
```python
# Initialize an existing Neptune run
run = neptune.init_run(
   with_id="NEP1-370",  # The run ID goes here
)
```
to be replaced with
```python
# Initialize an existing Neptune run
run = neptune.init_run(
    with_id="NEP1-8727",  # The run ID goes here
)
```
2.
```
        "feature_fraction": trial.suggest_uniform("feature_fraction", 0.2, 1.0),
        "bagging_fraction": trial.suggest_uniform("bagging_fraction", 0.2, 1.0),
```
to be replaced with
```
        "feature_fraction": trial.suggest_float("feature_fraction", 0.2, 1.0, step=0.1),
        "bagging_fraction": trial.suggest_float("bagging_fraction", 0.2, 1.0, step=0.1),
```

**Comment by SiddhantSadangi** [comment]
@Blaizzy - if you have a while, can you check this error:
`AttributeError: Can't get attribute 'IntDistribution' on <module 'optuna.distributions' from '/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/optuna/distributions.py'>`
This happens for all platforms on python3.11.

Sample job: https://github.com/neptune-ai/examples/actions/runs/3629421069/jobs/6121611625

**Review by normandy7 — DISMISSED** [review]

**Comment by SiddhantSadangi** [comment]
@Blaizzy - I've made some changes to the present cv results code so that it does not time out.
Can you check if things look good?

File: how-to-guides/present-cross-validation-results/notebooks/Neptune_cross_validation.ipynb [hunk 1]
```diff
@@ -282,13 +288,12 @@
...
    "outputs": [],
    "source": [
-    "trainset = datasets.CIFAR10(data_dir, transform=data_tfms[\"train\"], download=True)\n",
-    "\n",
-    "validset = datasets.CIFAR10(\n",
-    "    data_dir, train=False, transform=data_tfms[\"train\"], download=True\n",
-    ")\n",
-    "\n",
-    "dataset_size = len(trainset)"
+    "dataset = datasets.FakeData(\n",
```
Comment (Blaizzy) [inline how-to-guides/present-cross-validation-results/notebooks/Neptune_cross_validation.ipynb:291]: @SiddhantSadangi 

Could help me understand why are we using FakeData?

**Review by Blaizzy — CHANGES_REQUESTED** [review]

**Review by Blaizzy — APPROVED** [review]
LGTM!

**Comment by SiddhantSadangi** [comment]
@normandy7 - https://docs.neptune.ai/tutorials/tracking_cross_validation_results/#__tabbed_1_2 will also need to be updated.
Can you get the changes from the code?

**Comment by normandy7** [comment]
> @normandy7 - https://docs.neptune.ai/tutorials/tracking_cross_validation_results/#__tabbed_1_2 will also need to be updated. Can you get the changes from the code?

Yes, I think so. Would appreciate a review once published.

**Pull request merged**